#!/bin/sh -e
## 16_debian_email.dpatch by Eduard Bloch <blade@debian.org>
##
## DP: Define our contact address for program info

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
diff -urNad /home/inet/debian/tmp/cdrtools-2.01/cdrecord/cdrecord.c cdrtools-2.01/cdrecord/cdrecord.c
--- /home/inet/debian/tmp/cdrtools-2.01/cdrecord/cdrecord.c	2004-01-16 23:07:01.000000000 +0100
+++ cdrtools-2.01/cdrecord/cdrecord.c	2004-01-16 23:07:02.000000000 +0100
@@ -352,7 +352,7 @@
 								HOST_CPU, HOST_VENDOR, HOST_OS);
 
 #if	defined(SOURCE_MODIFIED) || !defined(IS_SCHILY_XCONFIG)
-#define	INSERT_YOUR_EMAIL_ADDRESS_HERE
+#define	INSERT_YOUR_EMAIL_ADDRESS_HERE "cdrtools@packages.debian.org"
 #define	NO_SUPPORT	0
 		printf("NOTE: this version of cdrecord is an inofficial (modified) release of cdrecord\n");
 		printf("      and thus may have bugs that are not present in the original version.\n");
diff -urNad /home/inet/debian/tmp/cdrtools-2.01/conf/xconfig.h.in cdrtools-2.01/conf/xconfig.h.in
--- /home/inet/debian/tmp/cdrtools-2.01/conf/xconfig.h.in	2004-01-16 23:06:46.000000000 +0100
+++ cdrtools-2.01/conf/xconfig.h.in	2004-01-16 23:07:39.000000000 +0100
@@ -532,8 +532,9 @@
  *
  * This restiction is introduced because this way, I hope that people
  * contribute to the project instead of creating branches.
- */
+*/
 #define	IS_SCHILY_XCONFIG
+#define SOURCE_MODIFIED
 /*
  * End restricted code for quality assurance.
  */
